airbyte-workers/src/main/java/io/airbyte/workers/protocols/singer/DefaultSingerTarget.java

```diff
@@ -67,9 +67,16 @@ public void start(StandardTargetConfig targetConfig, Path jobRoot) throws IOExce
     final JsonNode configDotJson = targetConfig.getDestinationConnectionImplementation().getConfiguration();
 
     IOs.writeFile(jobRoot, WorkerConstants.TARGET_CONFIG_JSON_FILENAME, Jsons.serialize(configDotJson));
+    IOs.writeFile(jobRoot, WorkerConstants.CATALOG_JSON_FILENAME, Jsons.serialize(targetConfig.getStandardSync().getSchema()));
 
     LOGGER.info("Running Singer target...");
-    targetProcess = pbf.create(jobRoot, imageName, "--config", WorkerConstants.TARGET_CONFIG_JSON_FILENAME).start();
+    targetProcess = pbf.create(
+        jobRoot,
+        imageName,
+        "write",
+        "--config",
+        WorkerConstants.TARGET_CONFIG_JSON_FILENAME,
+        "--catalog", WorkerConstants.CATALOG_JSON_FILENAME).start();
```

Comment:
Is it going to break the current singer targets?

Reply:
i need to check. i believe the contract is that this stuff will just be ignored as noise, but i need to check.